ceph_defaults/defaults/main.yml

```diff
@@ -3,8 +3,12 @@ ceph_origin: community
 ceph_dev_branch: main
 ceph_dev_sha1: latest
 ceph_rhcs_version: 5
+ceph_ibm_version: 5
 ceph_mirror: https://download.ceph.com
 ceph_stable_key: https://download.ceph.com/keys/release.asc
+ceph_community_repo_baseurl: "{{ ceph_mirror }}/rpm-{{ ceph_release }}/el{{ ansible_facts['distribution_major_version'] }}/"
+ceph_ibm_repo_baseurl: "https://public.dhe.ibm.com/ibmdl/export/pub/storage/ceph/{{ ceph_ibm_version }}/rhel{{ ansible_facts['distribution_major_version'] }}/"
+ceph_ibm_key: https://public.dhe.ibm.com/ibmdl/export/pub/storage/ceph/5/rhel8/ibm-key.asc
```

Comment:
@ktdreyer @andrewschoen do we know the exact url for that key?

Reply:
I don't know it, what's there now is a 404 for me though.

Reply:
We haven't yet pushed the repo content there. I believe we have enough control over the process to make it look like these paths

Reply:
It would be great to avoid `noarch` - that directory can be hard to keep synchronized with the arch-specific ones. Particularly if we're going to have some a complex mirror replication schedule